elements/bulbs-liveblog/bulbs-liveblog-responses.js

```diff
@@ -143,8 +143,8 @@ class BulbsLiveblogResponses extends BulbsHTMLElement {
       this.append(this.responseStaging.firstElementChild);
     }
     window.picturefill();
-    if (twttr) {
-      twttr.widgets.load();
+    if (twttr) {              // eslint-disable-line no-undef
```

Comment:
No we don't have anything describing that right now.

We run eslint. I think the GMG crew is starting to use http://editorconfig.org/ to wrangle editor configs, might be worth looking into.